Source/common/SNTCommonEnums.h

```diff
@@ -63,13 +63,15 @@ typedef NS_ENUM(NSInteger, SNTRuleState) {
 
   SNTRuleStateAllowCompiler = 5,
   SNTRuleStateAllowTransitive = 6,
+  SNTRuleStateAllowLocalBinary = 7,
+  SNTRuleStateAllowLocalSigningID = 8,
 };
 
 typedef NS_ENUM(NSInteger, SNTClientMode) {
   SNTClientModeUnknown,
-
   SNTClientModeMonitor = 1,
   SNTClientModeLockdown = 2,
+  SNTClientModeStandalone = 3,
```

Comment:
Not necessary for this PR but we should get rid of this enum at some point and use the one from the sync proto.